src/components/reservation/helper.ts

```diff
@@ -251,4 +253,56 @@ export const getInstantLoanBranchHoldingsAboveThreshold = (
     ({ materials }) => materials.length >= Number(instantLoanThresholdConfig)
   );
 
+export const removePrefixFromBranchId = (branchId: string) => {
+  const splitBranchId = branchId.split("-");
+  return splitBranchId[1];
+};
+
+export const getPatronAddress = (address: AddressV2) =>
+  `${address.street}, ${address.postalCode} ${address.city}`;
+
+export const translateOpenOrderStatus = (
+  status: SubmitOrderStatus,
+  t: UseTextFunction
+) => {
+  switch (status) {
+    case SubmitOrderStatus.OwnedAccepted:
+      return t("openOrderStatusOwnedAcceptedText");
+    case SubmitOrderStatus.AuthenticationError:
+      return t("openOrderAuthenticationErrorText");
+    case SubmitOrderStatus.BorchkUserBlockedByAgency:
+      return t("openOrderUserBlockedByAgencyText");
+    case SubmitOrderStatus.BorchkUserNotVerified:
+      return t("openOrderUserNotVerifiedText");
+    case SubmitOrderStatus.BorchkUserNoLongerExistOnAgency:
+      return t("openOrderUserNoLongerExistOnAgencyText");
+    case SubmitOrderStatus.InvalidOrder:
+      return t("openOrderInvalidOrderText");
+    case SubmitOrderStatus.NotOwnedIllLoc:
+      return t("openOrderNotOwnedIllLocText");
+    case SubmitOrderStatus.NotOwnedNoIllLoc:
+      return t("openOrderNotOwnedNoIllLocText");
+    case SubmitOrderStatus.NotOwnedWrongIllMediumtype:
+      return t("openOrderNotOwnedWrongIllMediumtypeText");
+    case SubmitOrderStatus.NoServicerequester:
+      return t("openOrderNoServicerequesterText");
+    case SubmitOrderStatus.OrsError:
+      return t("openOrderOrsErrorText");
+    case SubmitOrderStatus.OwnedOwnCatalogue:
+      return t("openOrderOwnedOwnCatalogueText");
+    case SubmitOrderStatus.OwnedWrongMediumtype:
+      return t("openOrderOwnedWrongMediumtypeText");
+    case SubmitOrderStatus.ServiceUnavailable:
+      return t("openOrderServiceUnavailableText");
+    case SubmitOrderStatus.UnknownError:
+      return t("openOrderUnknownErrorText");
+    case SubmitOrderStatus.UnknownPickupagency:
+      return t("openOrderUnknownPickupagencyText");
+    case SubmitOrderStatus.UnknownUser:
+      return t("openOrderUnknownUserText");
+    default:
+      return "";
+  }
```

Comment:
I think it makes more sense to use a map here, and the lookup if you have a definition in the map that matches the status.